app/models/ncr/work_order.rb

```diff
@@ -120,11 +122,21 @@ def current_approver_email_address
         self.currently_awaiting_approvers.first.email_address
       elsif self.approving_official
         self.approving_official.email_address
+      elsif self.emergency and self.approvers.empty?
+        EMERGENCY_APPROVER_EMAIL
```

Comment:
Right now this method is only used in the daily budget report email. I thought it would be important for readers to know why particular proposals were approved, since those approvals were done verbally rather than inside the system. If we don't care about communicating that, then having `nil` email address is fine with me.

@phirefly thoughts?